src/MediaCredits/index.js

```diff
@@ -24,13 +24,12 @@ const DivNavWrapper = styled.div`
   justify-content: space-between;
 `
 
-const NavUl = styled.ul`
```

Comment:
the ul was replaced because screen readers would read the number of buttons in it, which was often just one (disclaimer being hidden in esdp's). It seemed appropriate to move it to a div.